asllib/doc/Statements.tex

```diff
@@ -3421,4 +3421,12 @@ \subsubsection{Formally}
 \CodeSubsection{\SPragmaBegin}{\SPragmaEnd}{../Typing.ml}
 
 \subsection{Semantics}
-Pragmas are not associated with semantics.
+
+\subsubsection{Prose}
+Pragmas are structures present in the \untypedast \ which are designed to be used by
```

Comment:
We need a similar paragraph for global pragmas.

Reply:
I added a sentence linking back to the statement pragma semantics, since I don't really want to duplicate it, global pragmas don't seem to have their own section so I put it in the global declarations overview paragraph.

Reply:
I think this is resolved @Roman-Manevich could you please double check that you're happy with the solution.